src/component/toolbox/ToolboxView.ts

```diff
@@ -307,7 +313,7 @@ class ToolboxView extends ComponentView {
                     emphasisTextConfig.position = 'top';
                     needPutOnTop = true;
                 }
-                const topOffset = needPutOnTop ? (-5 - rect.height) : (itemSize + 8);
+                const topOffset = needPutOnTop ? (-5 - rect.height) : (itemSize + 10);
```

Comment:
This is to make the adjusted y aligns with the one not adjusted. The original value `8` seems not perfact.